test/ctest_helpers.cmake

```diff
@@ -37,6 +37,26 @@ function(create_test_executable target)
 
     # Define our test executable.
     add_executable(${target} ${CREATE_TEST_SOURCES})
+
+    # For MacOS builds we need to generate a dSYM bundle that contains the debug symbols for each 
+    # executable. The name of the binary will either be the name of the target or some other name
+    # passed to this function. We need to use the correct one for the dsymutil.
```

Comment:
Do we have a rough idea of how large the sum of the bundle files could be for all test executables? Just hope to make sure it's not going to cause disk space issues on the test host.

Reply:
Good question; the .dSYM bundle for the shared library and test format are about ~600 Bytes each so each bundle is small.

Reply:
Good question; the bundle size for the shared library is ~11MB and the bundle for test format (as an example) is 718KB so I think that's small enough.

Reply:
Good to know - that level of size is perfectly fine.